### PR TITLE
Bqml vertex model registry

### PR DESCRIPTION
**Pull request by soheilazangeneh**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Removed two repeated Vertex AI resources in the notebook objective section for clarity.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb [hunk 1]
```diff
@@ -32,18 +32,18 @@
         "<table align=\"left\">\n",
```
Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Use sentence case in title</p><p> </p><p>Vertex AI model registry -&gt; Vertex AI Model Registry</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>BQML -&gt; BigQuery ML</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>combine these 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>installs should be as a single pip install package1 package2 ....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Combine 2 text cells.</p><p>Remove Skip this step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>combine 2 code cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>BQML -&gt; BigQuery</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>I would combine these two code cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Is this specific to BQML models?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>place each param on separate line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (andrewferlitsch) [inline notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>combine 2 text cells</p><p>BQML -&gt; BigQuery ML</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Not directly. It's for users to confirm their model is uploaded to Vertex Model Registry.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: <p>Done.</p>

Comment (andrewferlitsch) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb:32]: ok, I will discuss offline how this differs from list(filter=....) or loading using the full model ID.

**Review by andrewferlitsch — APPROVED** [review]

File: notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb (hunk 1 above)
Comment (soheilazangeneh) [reply notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb]: Sounds good :)

**Comment by andrewferlitsch** [comment]
1 DISPLAY_NAME = "video_classification" + UUID
Step #5: ----> 3 job = aip.PipelineJob(
Step #5:       4     display_name=DISPLAY_NAME,
Step #5:       5     template_path="video_classification_pipeline.json",
Step #5:       6     parameter_values=parameters,
Step #5:       7     enable_caching=False,
Step #5:       8 )
Step #5:      10 job.run(service_account=SERVICE_ACCOUNT)
Step #5: 
Step #5: NameError: name 'aip' is not defined

**Pull request merged**